src/components/Sidebar/Sidebar.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "./sidebar.css";
+
+import { AiFillHome } from "react-icons/ai";
+import {
+  MdOutlineExplore,
+  MdOutlineHistory,
+  MdOutlineWatchLater,
+} from "react-icons/md";
+import { RiPlayList2Fill } from "react-icons/ri";
+import { NavLink } from "react-router-dom";
+
+export const Sidebar = () => {
+  return (
+    <aside className="aside">
+      <div className="aside-links">
+        <NavLink
+          to="/"
+          className={({ isActive }) => (isActive ? "active" : "")}
+        >
+          <AiFillHome />
+        </NavLink>
+
+        <NavLink
+          to="/explore"
+          className={({ isActive }) => (isActive ? "active" : "")}
+        >
+          <MdOutlineExplore />
+        </NavLink>
+
+        <NavLink
+          to="/watch-later"
+          className={({ isActive }) => (isActive ? "active" : "")}
+        >
+          <MdOutlineWatchLater />
+        </NavLink>
+
+        <NavLink
+          to="/playlist"
+          className={({ isActive }) => (isActive ? "active" : "")}
+        >
+          <RiPlayList2Fill />
+        </NavLink>
+
+        <NavLink
+          to="/history"
+          className={({ isActive }) => (isActive ? "active" : "")}
+        >
```

Comment:
instead of repeating the jsx for navlink you can create an array of links and map over them

Reply:
Yup, i'll do it. Thank you for the review